BaseTools/Plugin/BmpCheck/BmpCheck_plug_in.json

```diff
@@ -0,0 +1,5 @@
+{
+  "scope": "surfacefamily",
```

Comment:
Doesn't have to be done here, but we should consider making this agnostic of surface in the future.

Reply:
I'll knock this out after 202405 is released. The plugin needs an update or two also. Test3 doesn't actually test anything at the moment.